### PR TITLE
Add Machine Learning EBook in Indonesian Language

### PR DESCRIPTION
**Pull request by adimasmudi**
## What does this PR do?
Add Machine Learning Ebook in Indonesia Language and made a markdone little bit neat or clean

### Why is this valuable (or not)?
because the ability to understand a concept of machine learning algorithm is a must (i think) to become very good ML Engineer

### How do we know it's really free?
it's like the college book and published for public

### For book lists, is it a book? For course lists, is it a course? etc.
it's book

## Checklist:
- [ x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ x ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ x ] Include author(s) and platform where appropriate.
- [ x ] Put lists in alphabetical order, correct spacing.
- [ x ] Add needed indications (PDF, access notes, under construction).
- [ x ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LuigiImVector** [comment]
you changed the file structure, be careful with your IDE plugins and fix the problem

**Comment by adimasmudi** [comment]
> you changed the file structure, be careful with your IDE plugins and fix the problem

Yea, it happened because i used prettier. I've disabled that extension and fix the problem

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -127,6 +127,7 @@
 
 ### Machine Learning
 
+* [Pengenalan Konsep Pembelajaran Mesin dan Deep Learning](https://drive.google.com/file/d/1SxUcjunRR6roas5LUS5wyyy3-h7-oCxZ/view) - Jan Wira Gotama Putra (PDF)
```
Comment (LuigiImVector) [inline books/free-programming-books-id.md:130]: we don't accept drive link

File: books/free-programming-books-id.md [hunk 2]
```diff
@@ -180,7 +181,6 @@
 * [Dasar Pemrograman Python](https://www.pythonindo.com/tutorial-python-dasar/)
 * [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan *(Membutuhkan akun Google Play Books atau email yang valid)*
 * [Kursus Singkat Machine Learning dengan TensorFlow API](https://developers.google.com/machine-learning/crash-course?hl=id)
-* [Python Untuk Pemula](https://santrikoding.com/ebook/python-untuk-pemula) - Rizqi Maulana
```
Comment (LuigiImVector) [inline books/free-programming-books-id.md:183]: why did you remove it?
